src/main/kotlin/study/racingcar/entity/Car.kt

```diff
@@ -1,11 +1,14 @@
 package study.racingcar.entity
 
-class Car(var moveCount: Int = 0) {
+class Car(private var moveCount: Int = 0) {
+    val currentPosition: Int
+        get() = moveCount
+
     fun move(distance: Int) {
-        moveCount += distance
+        if (distance > 3) moveCount ++
     }
 
-    fun getResult(): String {
-        return "-".repeat(moveCount)
+    fun getResult(symbol:String): String {
+        return symbol.repeat(moveCount)
```

Comment:
도메인은 상태만 책임지도록 하는걸 권장드려요 출력은 UI에서 책임지도록 하는것이죠. 그럼 변환의 주체가 도메인이 아닌 UI로 바뀌게 됩니다. 

```suggestion
    fun getMoveCount(): Int = moveCount
    
    ...
class OutputView {
    ...
    
    fun view(racingHistories: List<RacingHistory>) {
        racingHistories.joinToString {
           val count = history.getMoveCount()
           
           SKID_MARK.repeat(count)
        }
    }
}
```

이런식으로 (예시용 수도 코드입니다) 도메인은 상태값만을 제공하고, 이를 가공하는건 UI에서 책임지는것이죠.